mitremap-notebook/requirements.txt

```diff
@@ -0,0 +1,146 @@
+adal==1.2.7
+aiohttp==3.8.3
+aiosignal==1.2.0
+anyio==3.6.1
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+asttokens==2.0.8
+async-timeout==4.0.2
+attrs==22.1.0
+azure-common==1.1.28
+azure-core==1.26.0
+azure-identity==1.11.0
+azure-mgmt-core==1.3.2
+azure-mgmt-subscription==3.1.1
+backcall==0.2.0
+beautifulsoup4==4.11.1
+bleach==5.0.1
+bokeh==2.4.3
+branca==0.5.0
+certifi==2022.9.24
+cffi==1.15.1
+charset-normalizer==2.1.1
+click==8.1.3
+cloudpickle==2.2.0
+colorama==0.4.5
+contourpy==1.0.5
+cryptography==38.0.2
+cycler==0.11.0
+debugpy==1.6.3
+decorator==5.1.1
+defusedxml==0.7.1
+Deprecated==1.2.13
+dnspython==2.2.1
+entrypoints==0.4
+executing==1.1.1
+fastjsonschema==2.16.2
+filelock==3.8.0
+folium==0.13.0
+fonttools==4.37.4
+frozenlist==1.3.1
+geoip2==4.6.0
+h11==0.12.0
+html5lib==1.1
+httpcore==0.15.0
+httpx==0.23.0
+idna==3.4
+importlib-metadata==5.0.0
+importlib-resources==5.10.0
+iocextract==1.13.1
+ipykernel==6.16.0
+ipython==8.5.0
+ipython-genutils==0.2.0
+ipywidgets==7.5.1
+isodate==0.6.1
+jedi==0.18.1
+Jinja2==3.1.2
+joblib==1.2.0
+jsonschema==4.16.0
+jupyter-core==4.11.1
+jupyter_client==7.4.2
+jupyterlab-pygments==0.2.2
+kiwisolver==1.4.4
+KqlmagicCustom==0.1.114.post22
+llvmlite==0.39.1
+lxml==4.9.1
+MarkupSafe==2.1.1
+matplotlib==3.6.1
+matplotlib-inline==0.1.6
+maxminddb==2.2.0
+mistune==2.0.4
+msal==1.20.0
+msal-extensions==1.0.0
+msrest==0.7.1
+msrestazure==0.6.4
+msticpy==2.1.2
```

Comment:
If you did want to cut this down a lot of this packages are dependencies of MSTICPy so you could choose to de-dupe if you wanted. Again not a need to do though.

Reply:
I'd be interested to know if there are some open source packages which do this cleanly! For now, it might be better to let the file be as is, given the strict dependencies for a couple packages I have mentioned in the jupyter notebook.